website/content/docs/ecosystem.mdx

```diff
@@ -6,194 +6,86 @@ description: Comparison between Nomad and Kubernetes
+- [Firecracker](https://www.nomadproject.io/plugins/drivers/community/firecracker-task-driver)
+- [Podman](https://www.nomadproject.io/plugins/drivers/podman)
+- [LXC](https://www.nomadproject.io/plugins/drivers/community/lxc)
 
-https://codefresh.io/docs/docs/yaml-examples/examples/nomad/
+## Application Definition & Image Build
 
-#### CircleCI
+- [Levant](https://github.com/hashicorp/levant)
+- [Packer](https://www.packer.io/)
+- [Waypoint](https://learn.hashicorp.com/collections/waypoint/get-started-nomad)
 
-https://circleci.com/docs/2.0/nomad/
+## Container Registry
 
-#### Drone
+- [JFrog Artifactory](https://jfrog.com/blog/cluster-management-made-simple-with-jfrog-artifactory-and-hashicorp-nomad/)
 
-https://docs.drone.io/runner/nomad/overview/
+## Observability and Analysis
 
-#### Jenkins
+- Prometheus
+  - [Using Prometheus to Monitor Nomad Metrics](https://learn.hashicorp.com/tutorials/nomad/prometheus-metrics)
+  - [Start Prometheus](https://learn.hashicorp.com/tutorials/nomad/dynamic-application-sizing?in=nomad/nomad-1-0#start-prometheus)
+- [Grafana](https://www.metricfire.com/blog/monitoring-hashicorp-nomad-with-prometheus-and-grafana/)
+- [DataDog](https://docs.datadoghq.com/integrations/nomad/)
+- [Turbonomics](https://www.youtube.com/watch?v=lwtIaPpdDsc)
+- [Circonus](https://www.hashicorp.com/integrations/circonus/nomad)
+- [Flowmill](https://www.flowmill.com/blog/announcing-flowmill-s-integration-with-nomad/)
+- [Sematext](https://www.hashicorp.com/integrations/sematext/nomad)
+- [Splunk](https://www.kmruddy.com/2020/deploying-splunk-enterprise-with-nomad/)
 
-https://plugins.jenkins.io/nomad/
+## Secret Management
 
-#### Buildkite
+- [Vault](https://www.nomadproject.io/docs/integrations/vault-integration)
 
-https://buildkite.com/works-with/hashicorp
+## Service Mesh
 
-### Container Runtime
+- [Consul](https://www.nomadproject.io/docs/integrations/consul-integration)
 
-#### Containerd
+## Provisioning
 
-https://www.nomadproject.io/plugins/drivers/community/containerd
+- [Terraform](https://registry.terraform.io/providers/hashicorp/nomad/latest/docs)
+- [Chef](https://github.com/nathwill/chef-nomad)
+- [Ansible](https://github.com/ansible-community/ansible-nomad)
 
-#### Windows IIS
+## Cloud Native Network
 
-https://github.com/Roblox/nomad-driver-iis
+- [CNI](https://www.nomadproject.io/docs/integrations/consul-connect#cni-plugins)
 
-#### Jailtask
+## Service Proxy
 
-https://www.nomadproject.io/plugins/drivers/community/jail-task-driver
+- [Envoy](https://www.nomadproject.io/docs/integrations/consul-connect)
+- [NGINX](https://learn.hashicorp.com/tutorials/nomad/load-balancing-nginx)
+- [Traefik](https://learn.hashicorp.com/tutorials/nomad/load-balancing-traefik)
 
-#### Pot
+## Storage
 
-https://github.com/trivago/nomad-pot-driver
+- [CSI](https://www.nomadproject.io/docs/concepts/plugins/csi)
+- [Portworx](https://www.hashicorp.com/integrations/portworx/nomad)
```

Comment:
Another dead link that drops us off at https://www.hashicorp.com/partners/find-a-partner?category=tech

Reply:
Want to remove this one too?

Reply:
I'm not really familiar with the partner integrations content, but I think if you search for the name in that "find-a-partner" it'll lead to something that looks right to me. For example, for this one: https://www.hashicorp.com/partners/tech/portworx#all

Reply:
I will update the link to be `https://learn.hashicorp.com/tutorials/nomad/stateful-workloads-portworx`!

Reply:
Updated in: https://github.com/hashicorp/nomad/pull/14549/commits/8f7444a681f8f685e46ab05bc96830fd0de05543